specification/timeseriesinsights/resource-manager/Microsoft.TimeSeriesInsights/2017-11-15/timeseriesinsights.json

```diff
@@ -323,6 +323,43 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.TimeSeriesInsights/environments/{environmentName}/status": {
+      "get": {
+        "operationId": "Environments_Status",
+        "x-ms-examples": {
+          "EnvironmentsBySubscription": { "$ref": "./examples/EnvironmentsStatusGet.json" }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/EnvironmentNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "description": "Gets the status of the environment, its associated operations like ingress.",
```

Comment:
At minimum, we need an 'and': 
>Gets the status of the environment, and its associated operations like ingress.

I'm not sure I like "operations" that much though, since it already has a context in resource management (e.g. ListOperations or "long-running operations"). What about
>Gets the status of the environment, and its internal state in the Time Series Insights service.